api/src/Microsoft.Azure.IIoT.OpcUa.Api.Registry/src/Models/DiscoveryMode.cs

```diff
@@ -18,6 +18,12 @@ public enum DiscoveryMode {
         [EnumMember]
         Off,
 
+        /// <summary>
```

Comment:
Please add to the end of the enum, or this will break backcompat since many enums are specified as int and this shifts the following enum values.

Reply:
added